pkg/model/config/config_test.go

```diff
@@ -37,11 +37,6 @@ var _ = Describe("Config", func() {
 			expectedConfig Config
 		)
 
-		By("Using config version 1")
```

Comment:
Should we keep this check to ensure that v1 usage continues to produce the expected behavior (failure?)